### ISSUE TITLE
The testing_acc1 of dvsgesture is only 80.

### ISSUE THREAD

**Issue by KoiLiu** [fangwei123456/Spike-Element-Wise-ResNet#11]
Hello, I trained the smodel in dvsgesture. But the testing_acc1 is around 80%, which is much lower than the accuracy in your paper. I used the Adam with lr=0.001 and the others are the default.
`python train.py --tb --amp --output-dir ./logs --model SEWResNet --connect_f ADD --device cuda:0 --lr-step-size 64 --epoch 192 --T_train 12 --T 16 --data-path ./dataset --lr 0.001 --adam`
The training loss and other results are as followed:
![AC75A908-D3FA-47AF-AA11-290D0EE2BC67](https://user-images.githubusercontent.com/47879103/165747477-d7c81132-0502-4e1c-ac9c-6191b744481d.jpeg)
![D0966B9A-A913-4F0B-8E96-4534BC86C87C](https://user-images.githubusercontent.com/47879103/165747504-cf1af540-4c7b-44aa-ba57-f9b69d727eb3.jpeg)
![43026CF9-CC4D-4527-84E0-CFFF8062C0EC](https://user-images.githubusercontent.com/47879103/165747509-83992425-410e-41fc-8cf6-f56e6096026a.jpeg)

**Comment by fangwei123456** [comment]
Refer to this: https://github.com/fangwei123456/Spike-Element-Wise-ResNet/issues/7#issuecomment-1065849077

**Comment by KoiLiu** [comment]
Thank for your reply. I will try it again.

**Comment by KoiLiu** [comment]
I git the latest version of SJ and retrained the train.py in the gesture. Maybe I didn't express my problem clearly before. I trained the model on the DVS128Gesture dataset, not the Imagenet. And this time the acc of test is only 81.94%, which is still much lower than 97.92 acc in your work. I don't know the reson for it. More details are as followed:
`Namespace(T=16, T_train=12, adam=True, amp=True, batch_size=16, connect_f='ADD', data_path='./dataset', device='cuda:0', dist_url='env://', distributed=False, epochs=192, lr=0.001, lr_gamma=0.1, lr_step_size=64, model='SEWResNet', momentum=0.9, output_dir='./logs', print_freq=64, resume='', start_epoch=0, sync_bn=False, tb=True, test_only=False, weight_decay=0, workers=4, world_size=1)...
Training time 0:50:47 max_test_acc1 81.94444444444444 test_acc5_at_max_test_acc1 100.0`

**Comment by fangwei123456** [comment]
[opt='SGD'](https://github.com/fangwei123456/Spike-Element-Wise-ResNet/blob/main/origin_logs/cifar10dvs/SEWResNet_ADD_T_16_T_train_None_SGD_lr_0.01_CosALR_64_amp/args.txt)

**Comment by fangwei123456** [comment]
[adam=False](https://github.com/fangwei123456/Spike-Element-Wise-ResNet/blob/main/origin_logs/dvsgesture/DVSGNetSEW32_b16_lr0.001_T16_Ttrain12_zi_steplr64_0.1_sgd_cnf_add_logs/args.txt)
